projects/timer_challenge/src/main.c

```diff
@@ -0,0 +1,37 @@
+#include <stdint.h>
+#include <stdlib.h>
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+#define HALF_SEC 500
+
+typedef struct Counters {
+  uint8_t counter_a;
+  uint8_t counter_b;
+} Counters;
+
+void prv_counter_increment(SoftTimerId id, void *context) {
+  Counters *counters = context;
+
+  counters->counter_a += 1;
```

Comment:
I sort of prefer counter_a++ but functionally it does the same thing in this code